expansion/src/main/java/no/unit/nva/expansion/model/ExpandedPublishingRequest.java

```diff
@@ -191,4 +192,32 @@ public PublishingWorkflow getWorkflow() {
     public void setWorkflow(PublishingWorkflow workflow) {
         this.workflow = workflow;
     }
-}
+
+    private static ExpandedTicketStatus getExpandedTicketStatus(PublishingRequestCase publishingRequestCase) throws NotFoundException {
+        switch (publishingRequestCase.getStatus()) {
+            case PENDING:
+                return getNewTicketStatus(publishingRequestCase);
+            case COMPLETED:
+                return ExpandedTicketStatus.COMPLETED;
+            case CLOSED:
+                return ExpandedTicketStatus.CLOSED;
+            default:
+                throw new NotFoundException ("Invalid PublishingRequestCase status " + publishingRequestCase.getStatus());
+        }
+    }
+
+    private static ExpandedTicketStatus getNewTicketStatus(PublishingRequestCase publishingRequestCase) {
+        if (isNull(publishingRequestCase.getAssignee())) {
+            return ExpandedTicketStatus.NEW;
+        } else {
+            return ExpandedTicketStatus.PENDING;
+        }
+    }
+
+    private TicketStatus getTicketStatus() {
+        if (!this.getStatus().equals(ExpandedTicketStatus.NEW)) {
+            return TicketStatus.parse(this.getStatus().toString());
+        }
+        return null;
```

Comment:
Same as above

Reply:
On second thought, if toTicketEntry() is only used in testing, move the code to the test. 
This so the production code shouldn't have functionality it doesn't need -> like decompressing after an expansion.

Reply:
I agree with the second thought